Source/Libraries/CorruptCore/MemoryDomains.cs

```diff
@@ -1729,9 +1729,9 @@ public override string ToString()
             return "Multiple Files";
         }
 
-        public string getCompositeFilename(string prefix)
+        public string getCompositeFilename()
```

Comment:
This seems like a bug to me. I'm not sure why these file prefixes aren't being used.

Reply:
The intention of this code is very simple. When corrupting files on the disk, we create backups of the files. The intention of that is so you can easily restore the clean file both before corrupt & on exiting the corruptor, loading another file, etc.

I don't even want to know how the code ended up in this state. I'm looking at this code and I'm just as confused as you are. It's an absolute mess and I can barely tell what's going on, but it does seem to be completely broken.  Some of this seems to have been copy-pasted out of the legacy Windows Glitch Harvester but not fully cleaned up.   


The key point here is the ToString() override on line 1117 https://github.com/ircluzar/RTCV/blob/master/Source/Libraries/CorruptCore/MemoryDomains.cs#L1117  
This code looks like someone replaced the prefix code and replaced it with this tostring trick, but didn't fully rip out the prefix code.  

This code 100% needs to be re-written as a generic module we can use to make &  use to keep track of file backups for various implementations. CemuStub actually has its own backup implementation.

Reply:
Thanks for that context. Going to mark this resolved since it's out of scope for this change (unless you want me to revert my changes here)